rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -27,13 +29,67 @@ local function fetch_request_body()
   return body
 end
 
+-- Returns the certificate and key for a given host
+function _M.get_cert_key(hostname)
```

Comment:
`get_pem_cert_key` to be consistent with controller and https://github.com/Shopify/ingress/pull/60/files